### PR TITLE
Update ReadME.txt with deepsparse-ent included

### PR DESCRIPTION
**Pull request by rgreenberg1**
Added in a proposed new ReadME for Deepsparse with refernces to the differences between deepsparse and deepsparse-ent as well as how to get started with enterprise. @bfineran and @markurtz plz review the section on deepsparse license getting set up with the env. variable. I was not sure of the detailed process here.

### CODE REVIEW

**Comment by rgreenberg1** [comment]
@markurtz and @bfineran check this out. Have some info to fill in with links, but comment on structure as well as plz add content on the deepsparse.license and validate license instructions.

File: README.md [hunk 1]
```diff
@@ -60,20 +60,104 @@ A CPU runtime that takes advantage of sparsity within neural networks to reduce
...
+
+The DeepSparse Enterprise Edition requires a valid license to run the engine and can be licensed for production, commercial applications. There are two options available:
+
+### 90-Day Enterprise Trial License
+To try out the DeepSparse Enterprise Edition and get a Neural Magic Trial License, complete our [registration form](https://neuralmagic.wpengine.com/deepsparse-engine-free-trial/). Upon submission, the license will be emailed immediately to you and your 90-day term starts right then.
+
+### Enterprise Edition License
+To learn more about DeepSparse Enterprise Edition pricing, [review our pricing page](https://neuralmagic.wpengine.com/deepsparse-engine-free-trial/). Details on how to [contact our Sales team](https://neuralmagic.com/deepsparse-engine/#form) and discuss your use case further for a custom quote are also available.
+
+## DeepSparse Enterprise Edition License Management
```
Comment (corey-nm) [inline README.md:113]: Maybe re-organize these sections as follow for better flow:

1. DeepSparse Enterprise Edition
    1. Getting a License
        1.  90-day Enterprise Trial License
        2. Enterprise Edition License
    2. Installing a license
    3. Validating a license

```markdown
## DeepSparse Enterprise Edition

### Getting a license
#### 90-day Enterprise Trial License
#### Enterprise Edition License

### Installing a license
### Validating a license
```

File: README.md [hunk 2]
```diff
@@ -60,20 +60,104 @@ A CPU runtime that takes advantage of sparsity within neural networks to reduce
...
+```
+
+`deepsparse.license` takes one argument that is a reference to the license and can be referenced in the `deepsparse.license` command as either `“license_string”` or `path_to_license`. 
+
+To initialize a license on a machine:
+1. Confirm you have deepsparse-ent installed in a fresh virtual environment. 
+	- Note: Installing deepsparse and deepsparse-ent on the same virtual environment could yield unsupported behaviors. 
+2. Take your license.txt file and move it to the directory you wish to use to run deepsparse-ent for inference. 
+3. You can set an environment variable to store the license to make future startup of the Deepsparse Enterprise Edition faster. 
+4. Run `deepsparse.license` with the `“license_string”` or `path_to_license` as an argument as follows:
```
Comment (corey-nm) [inline README.md:130]:
```suggestion
4. Run `deepsparse.license` with the `<license string>` or `path/to/license.txt` as an argument as follows:
```

File: README.md [hunk 3]
```diff
@@ -60,20 +60,104 @@ A CPU runtime that takes advantage of sparsity within neural networks to reduce
...
+	- `deepsparse.license “samplelicensetring”`
+	- `deepsparse.license ./license.txt`
+5. Once the license is authenticated, you should see a splash message indicating that you are now running DeepSparse Enterprise Edition. 
+6. Mark/Ben to help fill in how exactly the env variable gets set and how to do that.
```
Comment (corey-nm) [inline README.md:134]: Calling this out in comment

Comment (jeanniefinks) [reply README.md:134]: @bfineran see line 134 above.

**Review by bfineran — APPROVED** [review]

**Review by markurtz — APPROVED** [review]

**Pull request merged**